com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs

```diff
@@ -277,6 +303,10 @@ static void CheckPersistencyPopupAlreadyOpened()
         [Callbacks.DidReloadScripts]
         static void WizardBehaviour()
         {
+            // If the wizard does not need to be shown at start up, do nothing.
+            if (!HDProjectSettings.wizardIsStartPopup)
```

Comment:
@RSlysz, we do not need to do all the frame waiting if the wizard doesn't need to show on startup